apps/crn-server/test/data-providers/reminders.data-provider.test.ts

```diff
@@ -1146,6 +1147,299 @@ describe('Reminder Data Provider', () => {
+          ${`Grantee`} | ${`Lead PI (Core Leadership)`} | ${`not fetch`}
+          ${`Grantee`} | ${'Co-PI (Core Leadership)'}   | ${`not fetch`}
+          ${`Grantee`} | ${'Collaborating PI'}          | ${`not fetch`}
+          ${`Grantee`} | ${`Key Personnel`}             | ${`not fetch`}
+          ${`Grantee`} | ${`Scientific Advisory Board`} | ${`not fetch`}
+          ${`Grantee`} | ${`Project Manager`}           | ${`fetch`}
+          ${`Grantee`} | ${`ASAP Staff`}                | ${`not fetch`}
+          ${`Staff`}   | ${`Lead PI (Core Leadership)`} | ${`not fetch`}
+          ${`Staff`}   | ${`Lead PI (Core Leadership)`} | ${`not fetch`}
+          ${`Staff`}   | ${'Co-PI (Core Leadership)'}   | ${`not fetch`}
+          ${`Staff`}   | ${'Collaborating PI'}          | ${`not fetch`}
+          ${`Staff`}   | ${`Key Personnel`}             | ${`not fetch`}
+          ${`Staff`}   | ${`Scientific Advisory Board`} | ${`not fetch`}
+          ${`Staff`}   | ${`Project Manager`}           | ${`not fetch`}
+          ${`Staff`}   | ${`ASAP Staff`}                | ${`not fetch`}
+        `(
+          `Should $expected the reminder when user has asap role $asapRole and team role $teamRole`,
+          async ({ asapRole, teamRole, expected }) => {
+            // set current time to one minute after the end of the fixture event
+            jest.setSystemTime(
+              DateTime.fromISO('2022-01-01T10:01:00Z').toJSDate(),
+            );
+            const squidexGraphqlResponse = getSquidexRemindersGraphqlResponse();
+            squidexGraphqlResponse.queryEventsContents![0]!.flatData.startDate =
+              '2022-01-01T08:00:00Z';
+            squidexGraphqlResponse.queryEventsContents![0]!.flatData.endDate =
+              '2022-01-01T10:00:00Z';
+
+            squidexGraphqlResponse.queryEventsContents![0]!.flatData.speakers![0]!.team![0]! =
+              { id: 'team-id-3' };
+
+            squidexGraphqlResponse.findUsersContent!.flatData.teams = [
+              {
+                id: [{ id: 'team-id-3' }],
+                role: teamRole,
+              },
+            ];
+            squidexGraphqlResponse.findUsersContent!.flatData!.role! = asapRole;
+
+            squidexGraphqlResponse.queryResearchOutputsContents = [];
+            squidexGraphqlClientMock.request.mockResolvedValueOnce(
+              squidexGraphqlResponse,
+            );
+
+            const result = await reminderDataProvider.fetch(
+              fetchRemindersOptions,
+            );
+
+            if (expected === 'fetch') {
+              const expectedEventRecentlyEndedReminder =
+                getUploadPresentationReminder();
+              expectedEventRecentlyEndedReminder.data.endDate =
+                '2022-01-01T10:00:00Z';
+              expect(result).toEqual({
+                total: 1,
+                items: [expectedEventRecentlyEndedReminder],
+              });
+            } else {
+              expect(
```

Comment:
so if you are able to expect just the one reminder above, can we also not assert that zero items are returned here?

Reply:
We can’t because when an event has ended in the past 72h and the user is CMS admin there’s a publish material reminder, so `result.items.length` won’t come as 0 for some role combinations